collections/_priorities/assetio-04-fix-scene-inheritance.md

```diff
@@ -0,0 +1,19 @@
+---
+last_updated: "2024-11-30"
+category: assetio
+rank: 4
+created: "2024-11-30"
+stage:
+  # Must be dates (YYYY-MM-DD), empty if not there yet.
+  planning: ""
+  ongoing: ""
+  done: ""
+anchor: "fix-scene-inheritance"
+title: "Fix scene inheritance"
```

Comment:
```suggestion
anchor: "improve-scene-inheritance"
title: "Improve scene inheritance"
```
or
```suggestion
anchor: "make-scene-inheritance-more-robust"
title: "Make scene inheritance more robust"
```
No need to call something *broken* in a public-facing document like this IMO. Scene inheritance *does* work, it just doesn't work robustly in all scenarios.